WebRoot/js/zimbraAdmin/common/ZaApp.js

```diff
@@ -95,6 +95,57 @@ ZaApp.checkMyRight = function(targetType,targetBy,targetVal,right,attrs) {
 	
 }
 
+ZaApp.toggleLoadingStatusBar = function (instance, show, label) {
+	var elementId = instance._htmlElId;
+	var element;
+	if (elementId) {
+		element = document.getElementById(elementId);
+		var localButtonTree = element.querySelectorAll(".xform_button");
```

Comment:
Are only the button(s) of `xform_button` class the scope of the control? The function should be more generic if it is  defined in ZaApp. If it is difficult, target buttons or items should be enabled/disabled in a view/controller and this function should just show/hide the loading status bar.

Reply:
I'll review that with Saket later today. For this use case, .xform_button elements are all those set as type: _BUTTON_ within a form.

Reply:
I mean, is it okay not to disable other items like input field? Just need to disable buttons only?

Reply:
@k-kato this was reviewed with @dawoodshaikh and @joshualandsman today and it was agreed that all buttons and input fields should be disabled. I have updated the code to cover all items regardless of their class.
Note: the `ZaMsg.BUSY_TESTING` has changed to `ZaMsg.BUSY_VALIDATING`. Loading and Creating are kept since they are going to be used in ZCS-11944.

Reply:
I checked the latest (= 2nd) commit and it looks good. 
One more question. Will a pull down menu button be disabled as well? For example, "Region" in "Add New Volume" dialog. I don't think it is a button or an input field...

If it is not easy to handle it and/or it is not important to disable such a menu, we can address it on a separate ticket in the near future.

Reply:
It wouldn't be simple to disable those fields programmatically, but a workaround would be to set CSS `pointer-events: none` so that they are not clickable.